samples/ActionScript/FooBar.as

```diff
@@ -0,0 +1,35 @@
+// A sample for Actionscript.
+
```

Comment:
Thanks for the great samples. They're small and show a nice diverse range in how the language could be used.  Just one teeny weeny question: what is the source of these examples and what license covers them?

We don't need an explicit license added or comment, just clarification as we can't include any samples that comes from sources licensed under licenses that are _not_ in this list https://github.com/github/linguist/blob/master/vendor/licenses/config.yml#L1-L12

Reply:
As I wrote above, I made these samples after reading through some Actionscript getting started articles. I was initially going to copy paste the actual samples from the Actionscript documentation, but their documentation is licensed as CC BY-NC-SA 3, which I presume is not an acceptable license.

Could of course search through some Github projects to find some real properly licensed samples.

As for the Angelscript samples, one of which is written by me and available here: https://github.com/codecat/ssbd-payload It's currently not licensed, but if you need it to be licensed I can slap an MIT license on it and call it a day. :)

Not sure where @ifarbod got their sample from.

Reply:
Ditto here. 🙇